hack/run-all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Run all the important scripts in a sensible order,
+# handy for switching branches, etc
+wd="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+$wd/clean-deps.sh &&
+$wd/install-deps.sh &&
+echo '[build-assets]' &&
+grunt build &&
+echo '[test-assets]' &&
+grunt test &&
+grunt test-integration
+# $wd/test-integration-headless.sh &&
```

Comment:
which one of these you use depends on where you want to use this script, if you want it to be able to be run on an ec2 instance / as part of jenkins job then you need to use headless.  If this is just for devs running locally then the grunt version is sufficient.  Comment at the top of the file if thats what you decide so its clear it shouldn't be run in a non-desktop environment.